web/pdf_thumbnail_viewer.js

```diff
@@ -135,15 +135,16 @@ var PDFThumbnailViewer = (function PDFThumbnailViewerClosure() {
       this.thumbnails = [];
       this._pagesRotation = 0;
       this._pagesRequests = [];
+
+      var container = this.container;
+      while (container.hasChildNodes()) {
```

Comment:
`this.container.textContent = '';` -- I think it will be easier to read (we can prefix it with the comment about the intent)